google-cloud-pubsub-v1/lib/google/cloud/pubsub/v1/publisher/helpers.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Cloud
+    module PubSub
+      module V1
+        module Publisher
+          ##
+          # Additional configuration for the client.
+          # The max_send_message_length and max_receive_message_length settings were originally added in
+          # https://github.com/googleapis/google-cloud-ruby/pull/1513.
+          #
```

Comment:
Don't add documentation here, as it could replace the real class documentation, depending on which file Yard processes first.

In fact, it's probably easier just to:

```ruby
Google::Cloud::Pubsub::V1::Publisher::Client.configure do |config|
  # ...
```

Reply:
(Now I realize we can do the same thing in the vision helper, and so avoid having to `rubocop:disable Style/Documentation`.)